client/src/components/productDetails/SizeSelector.jsx

```diff
@@ -1,33 +1,72 @@
 import React from 'react';
 import QuantitySelector from './QuantitySelector.jsx';
+import AddToCart from './AddToCart.jsx';
 
 const { useEffect, useState } = React;
 
 function SizeSelector({ selectedStyle }) {
   const [skus, setSkus] = useState({});
   const [selectedSizeAmount, setSelectedSizeAmount] = useState(0);
+  const initialSelection = { size: '', quantity: '' };
+  const [selection, setSelection] = useState(initialSelection);
 
   useEffect(() => {
     if (selectedStyle) {
-      console.log('selectedStyle', selectedStyle);
       setSelectedSizeAmount(0);
       setSkus(selectedStyle.skus);
+      setSelection(initialSelection);
     }
   }, [selectedStyle]);
 
   if (skus) {
     const sizesAndAmount = Object.values(skus);
     const skuSizes = Object.keys(skus);
 
+    const handleSizeSelection = (e) => {
+      const currentSku = skuSizes[e.target.value];
+      console.log('skuObj', skus[currentSku]);
```

Comment:
im sure youre still testing but don't forget to delete the console.logs later on =)